Snowflake.Data.Tests/SFConnectionPoolT.cs

```diff
@@ -504,6 +541,98 @@ public void TestConcurrentConnectionPoolingAsync()
             ConcurrentPoolingAsyncHelper(connStr, true);
         }
 
+        [Test]
+        public void TestRollbackTransactionOnPooledWhenExceptionOccurred()
+        {
+            SnowflakeDbConnectionPool.SetMaxPoolSize(1);
+
+            object firstOpenedSessionId = null;
+            using (var connection = new SnowflakeDbConnection())
+            {
+                connection.ConnectionString = ConnectionString;
```

Comment:
just curious why we cannot pass connection string in `SnowflakeDbConnection` constructor?